### DISCUSSION TITLE
What is the purpose of this repository?

### DISCUSSION THREAD

**Discussion by itmayziii** [standard/eslint-config-standard-with-typescript#1532] [Q&A] [answered]
A big reason I'm using eslint-config-standard-with-typescript is my company uses the [standard style guide.](https://standardjs.com/) 

It seems based on [this issue](https://github.com/standard/standard/issues/1957) on the standard repository that this repository is likely to deviate from the standard specification for non TypeScript code:

> -with-typescript will depart from its dependency on eslint-config-standard and therefore depart from its relationship with Standard. This would permit -with-typescript the freedom to introduce such changes that would currently be incompatible with standard. For example, new core ESLint (non-TypeScript) rule.

If this repository is not going to follow the standard style guide then what is the purpose of it? Seems the name was changed to `eslint-config-love` so I'm assuming anybody that was relying on this package for the standard style guide with TypeScript should look elsewhere? It's just very hard to determine the intent of this package as it stands. The README says `A TypeScript ESLint config that loves you` and I don't know what that means.

**Comment by mightyiam** [comment, marked as answer]
That's a good one. 

I should probably describe the attitude of rule curation.

Sorry. Yet another, albeit important task.

And yes, this has no longer any commitment to standard. In fact, style rules will be removed and possibly replaced by a formatter. Haven't looked into that yet.

**Reply by josephdpurcell** [reply]
I was looking for a quick opinionated eslint config and was struggling. Then I found this `eslint-config-love` and I GREATLY appreciate the simplicity.

I was surprisingly unaware of https://standardjs.com/. Looks great, but it doesn't use eslint so I would need to figure that out.

One purpose that stands out to me, then, is if you want a quick opinionated eslint config you've got it right here! I am doing a quick project so I need to hurry and I'm going to continue using this project. That said, I second @itmayziii's comment about aligning with an already published standard would make sense.

**Reply by mightyiam** [reply]
I'm glad you find value, @josephdpurcell. This project's relationship with Standard JS is history. As is Standard JS, itself, it seems.
